client/verta/verta/client.py

```diff
@@ -640,7 +640,7 @@ def get_or_create_experiment_run(self, *args, **kwargs):
         """
         return self.set_experiment_run(*args, **kwargs)
 
-    def get_or_create_registered_model(self, name=None, desc=None, labels=None, workspace=None, public_within_org=None, visibility=None, id=None):
+    def get_or_create_registered_model(self, name=None, desc=None, labels=None, workspace=None, public_within_org=None, visibility=None, id=None, actionType=None, dataType=None):
```

Comment:
In Python, everything should be `snake_case`
```suggestion
    def get_or_create_registered_model(self, name=None, desc=None, labels=None, workspace=None, public_within_org=None, visibility=None, id=None, action_type=None, data_type=None):
```

Reply:
done